composer.json

```diff
@@ -113,12 +117,16 @@
         "drupal/menu_link_weight": "^1.0-beta3",
         "drupal/menu_position": "~1.0@alpha",
         "drupal/metatag": "~1.3",
+        "drupal/migrate_file": "^1.1",
+        "drupal/migrate_plus": "^5.1",
+        "drupal/migrate_tools": "^5.0",
```

Comment:
Do we need all these if we depend on stanford_migrate? I think you could pull them out of here.